### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jaredsburrows**
* Updated broken link
* Updated example for build.gradle
* Updated robolectric from 2.3 to 2.4

### CODE REVIEW

**Comment by jaredsburrows** [comment]
This broke because Travis couldnt download a dependency.

**Comment by jaredsburrows** [comment]
I kicked off the build again and all is well.

File: README.md [hunk 1]
```diff
@@ -32,7 +32,13 @@ apply plugin: 'robolectric'
 
 Add test-only dependencies using the `androidTestCompile` configuration:
 ```groovy
-androidTestCompile 'org.robolectric:robolectric:2.3'
+androidTestCompile('junit:junit:4.12') {
+	exclude module: 'hamcrest-core'
```
Comment (ChristianBecker) [inline README.md]: I get the following `ClassNotFoundException` if I exclude the transitive `hamcrest-core` dependecy: `java.lang.NoClassDefFoundError: org/hamcrest/SelfDescribing`.

Comment (jaredsburrows) [inline README.md]: I was going to add in the `hamcrest` matchers but that is just my personal preference. Thank you for pointing that out.

**Pull request merged**